plugins/kotlin/idea/tests/test/org/jetbrains/kotlin/idea/editor/quickDoc/PackageQuickDocTest.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2000-2023 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.kotlin.idea.editor.quickDoc
```

Comment:
this test targets K1 plugin, it would be better to make it shared for both K1 and K2.

Reply:
Can you clarify how this targets K1? I don't see any references to K1 or K2 anywhere - I must be missing something.

Reply:
actually, two plugins are built from `plugins/kotlin/` codebase - for K1 and for K2. 

They share some common code, but some parts are completely different. 
`plugins/kotlin/idea/tests/test/` are tests for K1. You can see that some modules have `-k2` in their name, or shared if it is used for both plugins. 

In some cases, K2 test utilizes K1 test (both logic and test data) but specifies that it is K2-version:  for instance, you can compare `AbstractAutoImportTestBase` and `AbstractK2AutoImportTest`

Reply:
Ahh, OK. So do I just need to move this test to a different module? Which module would you suggest?